### ISSUE TITLE
Clang C++20 Feature: P1703R1 - Recognizing Header Unit Imports Requires Full Preprocessing

### ISSUE THREAD

**Issue by tstellar** [llvm/llvm-project#54043]
---
id: P1703R1
paper: https://wg21.link/p1703r1

**Comment by JVApen** [comment]
Status page indicates "Subsumed by P1857", which is already tracked via https://github.com/llvm/llvm-project/issues/54047 To be closed?

**Comment by ChuanqiXu9** [comment]
> Status page indicates "Subsumed by P1857", which is already tracked via #54047 To be closed?

Yeah... And from latest wg21 information, it look like P1703 may be discarded. So when P1857 got completed, this issue should not be closed. I guess the best solution may close this issue after the latest formal paper coming from the wg21.

**Comment by frederick-vs-ja** [comment]
> I guess the best solution may close this issue after the latest formal paper coming from the wg21.

Hmm... Is there's already such a latest formal paper now?
